app/assets/javascripts/comments.coffee

```diff
@@ -0,0 +1,20 @@
+class Comment
+  constructor: ->
+    @bindEvents()
+
+  bindEvents: =>
+    $(document).on "ajax:success", "[data-behavior='comment_destroy']",
+    @destroyComment
+    $(document).on "ajax:success", "[data-behavior='comment-form']",
+    @addComment
+
+  destroyComment: (event) =>
+    comment_id = $(event.currentTarget).data("id")
+    $(".comment-wrapper[data-id='#{comment_id}']").remove()
+
+  addComment: (event, data, status, xhr) =>
+    $(".comments").append xhr.responseText
+    $(".comment-form").find("#comment_text").val('')
+
+$ ->
+Comment = new Comment
```

Comment:
here is not the best place to initialize this object.

Or move this initialization to main file, or keep here but check if you are on comment page, to prevent creation of this object at all site pages